cdap-proto/src/main/java/co/cask/cdap/proto/MetadataSearchResultRecord.java

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright © 2015 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package co.cask.cdap.proto;
+
+import com.google.common.base.Objects;
+
+/**
+ * Represent the Metadata search result record.
+ */
+public class MetadataSearchResultRecord {
+  private final Id.NamespacedId id;
+  private final String type;
```

Comment:
Additional information should not hurt in my opinion. Just in case later we allow any type search.